README.md

```diff
@@ -454,6 +462,20 @@ Specifies an ID which is used to prevent resource recreation when the order in
 the list of projects changes.
 Default is `name`.
 
+#### [`webhook`](#webhooks-configuration) Object Attributes
+- **`name`**: *(Optional `string`)*
+The type of the webhook. `web` is the default and the only option.
+
+- **`active`**: *(Optional `bool`)*
+Indicate if the webhook should receive events. Defaults to `true`.
+
+- **`events`**: ***(Required `list(string`)***
+A list of events which should trigger the webhook. [See a list of available events.](https://developer.github.com/v3/activity/events/types/)
+
+- **`configuration`**: ***(Required `map(string)`)***
+A key/value pair of configuration for this webhook. Available keys are `url`, `content_type`, `secret` and `insecure_ssl`. `secret` is the shared secret, see [API documentation](https://developer.github.com/v3/repos/hooks/#create-a-hook).
```

Comment:
from the API docs: in case you want to implement the suggestion of removing configuration completely replacing it with just the values...

Name | Type | Description
-- | -- | --
url | string | Required. The URL to which the payloads will be delivered.
content_type | string | The media type used to serialize the payloads. Supported values include json and form. The default is form.
secret | string | If provided, the secret will be used as the key to generate the HMAC hex digest value in the X-Hub-Signature header.
insecure_ssl | string | Determines whether the SSL certificate of the host for url will be verified when delivering payloads. Supported values include 0 (verification is performed) and 1 (verification is not performed). The default is 0. We strongly recommend not setting this to 1 as you are subject to man-in-the-middle and other attacks.

Reply:
That does make the docs cleaner and more useful, huh?  I'm in 🙃